python/tskit/trees.py

```diff
@@ -5370,6 +5370,21 @@ def genealogical_nearest_neighbours(self, focal, sample_sets, num_threads=0):
                 arrays = pool.map(worker, splits)
             return np.vstack(list(arrays))
 
+    def kc_distance(self, other, lambda_=0.0):
+        """
+        Returns an analog to the ``kc_distance`` between trees but for two tree
```

Comment:
Might be better to be more direct here. Maybe "Returns the average :meth:`.Tree.kc_distance` between pairs of trees along the sequence. The average is weighted by the fraction of sequence length that each pair of trees overlap."

Or something like that.